doc/AsyncClient.md

```diff
@@ -45,12 +45,12 @@ This doc provides instructions on how to use async Java client to connect Nebula
           LOGGER.info(String.format("No Space Founded"));
         }
       }
-    
+
       @Override
       public void onFailure(Throwable throwable) {
         LOGGER.error("List Spaces Error");
       }
     }, service);
     ```
 
-  - Other functions including ```getPartsAlloc()```, ```listTags()``` and ```listEdges()``` take a specific String of space name or Integer of space Id to proceed. The return type are all ListenableFuture of response of the corresponding function type.
\ No newline at end of file
+  - Other functions including ```getPartsAlloc()```, ```listTags()``` and ```listEdges()``` take a specific String of space name or Integer of space ID to proceed. The return type are all ListenableFuture of response of the corresponding function type.
```

Comment:
The return type are all ListenableFuture of response of the corresponding function type.
 I don't really understand this sentence. Please help. 
@critical27